kasa/discover.py

```diff
@@ -800,42 +812,56 @@ def _get_device_instance(
             _LOGGER.debug("[DISCOVERY] %s << %s", config.host, pf(data))
         device = device_class(config.host, protocol=protocol)
 
-        di = discovery_result.get_dict()
+        di = discovery_result.to_dict()
         di["model"], _, _ = discovery_result.device_model.partition("(")
         device.update_from_discover_info(di)
         return device
 
 
-class EncryptionScheme(BaseModel):
+class _BaseMixin(DataClassJSONMixin):
```

Comment:
I'm wondering if we should split the models into their own module, what do you think?

Reply:
At the moment they're contained where they're used and I probably prefer that but don't feel strongly. Is it the `_BaseMixin` that causing you to suggest that? We might not be using the same base everywhere.

Reply:
Nah, just wondering if we should extract discovery internals into a separate package while we are doing cleanups, but it's good as it is. Maybe consider renaming to something like Discovery(Base)Mixin to make its usage clear?